api/lanes.py

```diff
@@ -1051,13 +1057,62 @@ def order_facet_to_database_field(cls, order_facet):
         else:
             return Facets.order_facet_to_database_field(order_facet)
 
+
+class CrawlableCollectionBasedLane(DynamicLane):
+
+    MAX_CACHE_AGE = 60 * 60 * 12 # 12 hours
```

Comment:
Should this be configurable? Is it only used when there's no configuration setting for the paged OPDS feed cache time?

Reply:
After taking a closer look at the CachedFeed code, I think the paged OPDS feed cache time is going to be used instead of MAX_CACHE_AGE, because it's going to be fetched as a 'page' type feed. Since my time is limited I'm going to take out this code and say that properly caching these feeds is a future piece of performance work, like speeding up the query.

Reply:
Maybe they should be separately configurable but the cache time for this feed should definitely be a lot longer than for a normal feed shown to patrons.